CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   due to the lack of a generated executable.  
   [Samuel Giddins](https://github.com/segiddins)
 
+* Print better promote message when unable to find a specification.
```

Comment:
nit: please add two missing spaces after the period for formatting.

Reply:
Done.